src/tests/Microsoft.VisualStudio.Composition.Tests/EmbeddedableTypesMixedTests.cs

```diff
@@ -20,7 +20,8 @@ namespace Microsoft.VisualStudio.Composition.Tests
     /// </summary>
     public class EmbeddedableTypesMixedTests
     {
-        [MefFact(CompositionEngines.V1Compat, "Microsoft.VisualStudio.Composition.NonEmbeddingTypeReceiver", typeof(EmbeddableTypesTests.PartThatExportsIVsProjectReference))]
+        [MefFact(CompositionEngines.V1Compat, "Microsoft.VisualStudio.Composition.NonEmbeddingTypeReceiver", typeof(EmbeddableTypesTests.PartThatExportsIVsProjectReference), SkipOnMono = true)]
+        [Trait("NoPIA", "SkipOnMono")]
```

Comment:
I'd prefer the "NoPIA" trait have a value of "true" rather than "SkipOnMono" since the value doesn't mean anything -- and the way you have it here, it looks like the value is somehow significant.

What I was hoping for was two traits:
```csharp
[Trait("NoPIA", "true")]
[Trait("SkipOnMono", "true")]
```

And then have the test discovery system skip on mono based on the second of these two.